hugegraph-core/src/main/java/com/baidu/hugegraph/backend/query/ConditionQuery.java

```diff
@@ -308,7 +311,7 @@ public String userpropValuesString(List<Id> fields) {
             boolean got = false;
             for (Relation r : this.userpropRelations()) {
                 if (r.key().equals(field) && !r.isSysprop()) {
-                    E.checkState(r.relation == RelationType.EQ,
+                    E.checkState(r.relation == RelationType.EQ || r.relation == RelationType.CONTAINS,
```

Comment:
don't exceed 80 chars in each line, reference [code style](https://github.com/hugegraph/hugegraph-doc/wiki/HugeGraph%E4%BB%A3%E7%A0%81%E9%A3%8E%E6%A0%BC%E6%8C%87%E5%8D%97)